.github/workflows/test-postgresql.yml

```diff
@@ -6,30 +6,33 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        entry:
-          - { ruby: 2.6.2, postgresql: 11 }
-          - { ruby: 3.1.1, postgresql: 11 }
-          - { ruby: 3.1.1, postgresql: 14 }
-    name: test (ruby=${{ matrix.entry.ruby }}, postgresql=${{ matrix.entry.postgresql }})
+        ruby: ['2.6.2', '3.1.1']
+        postgresql: [11, 14]
+        active_record: ['~> 6.0.0', '~> 6.1.0', '~> 7.0.0']
+        exclude:
+          - ruby: '2.6.2'
+            active_record: '~> 7.0.0'
+    name: test (ruby=${{ matrix.ruby }}, postgresql=${{ matrix.postgresql }}, active_record=${{ matrix.active_record }})
```

Comment:
I'd prefer it if we kept the original entry matrix format to be consistent with other .yml, and not use exclude because this is 2 x 2 x 3 and will grow out of control fast every time we add a new ruby version or a new AR version or a new PGsql version. The list of excludes becomes hard to manage. Add the active_record values that should have been there in the first place?